llama-index-integrations/llms/llama-index-llms-mistralai/pyproject.toml

```diff
@@ -27,7 +27,7 @@ exclude = ["**/BUILD"]
 license = "MIT"
 name = "llama-index-llms-mistralai"
 readme = "README.md"
-version = "0.1.6"
+version = "0.1.7"
```

Comment:
Just realized I updated the mistralai, meaning my changes were not published 🤦‍♂️

Reply:
Resolved in #12013